code/game/objects/items/tools/crowbar.dm

```diff
@@ -42,13 +42,26 @@
 	usesound = 'sound/weapons/sonic_jackhammer.ogg'
 	icon_state = "crowbar"
 	toolspeed = 0.1
+	w_class = WEIGHT_CLASS_SMALL
 	origin_tech = "combat=4;engineering=4;abductor=3"
 
+/obj/item/crowbar/small
+	name = "miniature titanium crowbar"
+	desc = "A tiny, lightweight titanium crowbar. It fits handily in your pocket."
+	force = 3
+	w_class = WEIGHT_CLASS_SMALL
+	throwforce = 3
+	materials = list(MAT_TITANIUM = 250)
+	icon_state = "crowbar_titanium"
+	item_state = "crowbar_titanium"
+	origin_tech = "materials=2"
+	toolspeed = 1.25
+
 /obj/item/crowbar/large
 	name = "crowbar"
```

Comment:
Definitely needs something, `heavy crowbar` maybe? It does 3x more damage than the regular one.